radis/spectrum/spectrum.py

```diff
@@ -3035,6 +3036,25 @@ def is_optically_thin(self):
                 + "value manually with s.conditions['self_absorption']=..."
             )
 
+    def print_struct(self):
```

Comment:
- You could create a method at the SpectrumFactory level that calls the latest profiler with `print_struct` ? Like `sf.print_perf_profile` . 

- maybe also in the Spectrum object itself. `Soectrum.print_perf_profile`

- Also add a small mention under ["Performance" section in the docs](https://radis.readthedocs.io/en/latest/lbl/lbl.html#performance), for instance : 

> Monitor performances
> -------------------------
> 
> Radis allows you to profile the spectrum calculation so you can further optimize it. Use :py:meth:`~radis.lbl.factory.print_perf_profile` to print the time spent on each step of the calculation. Typical output :: 
>  XXX  
> Results are also stored in a Spectrum object so you can compare them a posteriori. Use :py:meth:`~radis.spectrum.spectrum.Spectrum.print_perf_profile`. Calculation details are also shown during the calculation itself if using a sufficiently high verbose level (e.g. : `verbose=3`)